en/flying/basic_flying.md

```diff
@@ -55,6 +53,32 @@ The vehicle may disarm if you take too long to take off after arming (tune the t
 The [Failure Detector](../config/safety.md#failure-detector) will automatically stop the engines if there is a problem on takeoff.
 :::
 
+### Fixed-wing Takeoff
+
+Automatic takeoff using [Takeoff mode (FW)](../flight_modes_fw/takeoff.md) is highly recommended for fixed-wing vehicles!
+You can either [hand-launch](../flight_modes_fw/takeoff.md#catapult-hand-launch) (default) or use [runway takeoff](../flight_modes_fw/takeoff.md#runway-takeoff) (if supported by hardware and configured).
+
+For runway takeoff:
+
+1. Place the vehicle facing the planned takeoff direction (ideally facing the wind)
+1. Activate [Takeoff mode (FW)](../flight_modes_fw/takeoff.md)
+1. [Arm the vehicle](#arm) using an [RC switch](../config/flight_mode.md) or ground station
+
+   The vehicle will ramp up motors, and fly in the indicated direction, until it reaches the (parameter set) clearance height, then enter Hold mode.
+
+
+For catapult/hand-launch:
+
+1. Point the vehicle in the direction you want it to take off, in order to set the course (ideally facing the wind)
+1. Activate [Takeoff mode (FW)](../flight_modes_fw/takeoff.md)
+1. [Arm the vehicle](#arm) using an [RC switch](../config/flight_mode.md) or ground station
+1. Throw/launch the vehicle in the direction you want it to take off.
+
+   Motors will start after the launch is detected, after which the behaviour is the same as for runway takeoff.
+
+For more information see [Takeoff mode (FW)](../flight_modes_fw/takeoff.md).
+Using a takeoff item defined in [a mission plan](../flight_modes/mission.md#fw-mission-takeoff) is also recommended.

```

Comment:
The manual flying section was very MC specific. I've started building it out with FW stuff as well. My plan is to split this too and put it into MC and FW sections.

I might have put too much here on takeoff, or too little. Thoughts?

Reply:
too much, see comments above.